src/main/scala/com/typesafe/sbt/site/sphinx/SphinxPlugin.scala

```diff
@@ -56,15 +57,16 @@ object SphinxPlugin extends AutoPlugin {
   }
 
   def defaultVersionProperties(version: String) = {
-    val binV = CrossVersion.binaryVersion(version, "")
+    val binV = CrossVersion.binarySbtVersion(version)
     Map("version" -> binV, "release" -> version)
   }
 
   def installPackagesTask = Def.task {
     val runner = sphinxRunner.value
     val packages = sphinxPackages.value
     val s = streams.value
-    packages map { p => runner.installPackage(p, target.value, s.log) }
+    val t = target.value
```

Comment:
[`target` is SettingKey](https://github.com/sbt/sbt/blob/v1.0.0/main/src/main/scala/sbt/Keys.scala#L142) but https://github.com/sbt/sbt/issues/3299#issuecomment-321612015